src/base-lib/src/constant/constant.ts

```diff
@@ -406,6 +406,10 @@ export const QUICKSIGHT_DATASET_INFIX= '-dataset-';
 export const SCAN_METADATA_WORKFLOW_PREFIX = 'ScanMetadataWorkflow';
 export const CLICKSTREAM_SEGMENTS_WORKFLOW_PREFIX = 'ClickstreamUserSegmentsWorkflowStateMachine';
 export const CLICKSTREAM_SEGMENTS_CRON_JOB_RULE_PREFIX = 'Clickstream-SegmentJobRule-';
+export const REFRESH_MATERIALIZED_VIEWS_WORKFLOW_PREFIX = 'RefreshMaterializedViewsWorkflow';
+export const REFRESH_SP_STEP = 'REFRESH_SP';
```

Comment:
Will it be shared by multiple components?

Reply:
Good point, they are not shared to other component. Move them to analytics contant